### PR TITLE
Added TRDTrackQC parameters

### PR DESCRIPTION
**Pull request by f3sch**
As preliminary parameters for my thesis, I added some Track parameters to the QC workflow.
I need the charges for each layer, the momentum, the number of layers in which matching tracklets have been found.
Additionally I will start my analysis using the TPC PID, but will later look at specific decays.

Also I ran clang-format on the relevant files

### CODE REVIEW

File: Detectors/TRD/qc/src/Tracking.cxx [hunk 1]
```diff
@@ -55,9 +55,12 @@ void Tracking::checkTrack(const TrackTRD& trkTrd, bool isTPCTRD)
   TrackQC qcStruct;
   qcStruct.type = isTPCTRD ? 0 : 1;
   qcStruct.nTracklets = trkTrd.getNtracklets();
+  qcStruct.nLayers = trkTrd.getNlayersFindable();
```
Comment (martenole) [inline Detectors/TRD/qc/src/Tracking.cxx:58]: Note that "number of associated layers of a track" is actually the number of layers in which the track extrapolation was inside the geometrical acceptance of the TRD. It does not mean that a tracklet was added. This is the `nTracklets` variable

**Review by martenole — DISMISSED** [review]
Maybe you can make the comment on nLayers a bit more precise, otherwise this looks good

**Comment by f3sch** [comment]
I added a better description, requesting new review.

**Review by martenole — DISMISSED** [review]

**Review by bazinski — DISMISSED** [review]

**Comment by martenole** [comment]
Hi Felix, could you squash your commits into a single one and then force-push again? This will trigger a restart of the CI which failed due to reasons unrelated to your changes

**Comment by martenole** [comment]
> Hi Felix, could you squash your commits into a single one and then force-push again? This will trigger a restart of the CI which failed due to reasons unrelated to your changes

Hi @f3sch could you please do the squash? E.g. with `git rebase -i HEAD~4` and then create only a single commit with a short description of your changes

File: Detectors/TRD/qc/include/TRDQC/Tracking.h [hunk 2]
```diff
@@ -75,6 +74,8 @@ struct TrackQC {
   std::array<int, constants::NLAYER> trackletMcm{};                                      ///< the MCM number of the tracklet
   std::array<float, constants::NLAYER> trackletChi2{};                                   ///< estimated chi2 for the update of the track with the given tracklet
   std::array<std::array<int, constants::NCHARGES>, constants::NLAYER> trackletCharges{}; ///< charges of tracklets
+  std::array<float, constants::NLAYER> trackletPhi{};         //< Phi 0:2Pi value of Tracklet
```
Comment (martenole) [inline Detectors/TRD/qc/include/TRDQC/Tracking.h]: @f3sch this should be `trackPhi` and `trackEta`. Could you please add it to the paragraph with the track parameters?

**Review by martenole — APPROVED** [review]

**Pull request merged**